cpp/dolfinx/fem/interpolate.h

```diff
@@ -835,103 +841,81 @@ void interpolate(Function<T>& u, std::span<const T> f,
   }
 }
 
-namespace impl
-{
-
-/// Interpolate from one finite element Function to another
-/// @param[in,out] u The function to interpolate into
-/// @param[in] v The function to be interpolated
-/// @param[in] cells List of cell indices to interpolate on
+/// Generate data needed to interpolate discrete functions across different
+/// meshes
+///
+/// @param[out] u The function to interpolate into
+/// @param[in] v The function to interpolate from
+/// @param[in] cells Indices of the cells in the mesh on which to
```

Comment:
Specifically refer to whether these are cells associated with mesh on function u or v.